src/main/scala/coreplex/Coreplex.scala

```diff
@@ -65,21 +80,82 @@ abstract class Coreplex(implicit val p: Parameters, implicit val c: CoreplexConf
 }
 
 class DefaultCoreplex(tp: Parameters, tc: CoreplexConfig) extends Coreplex()(tp, tc) {
+  class TileWrapper(tileBuilder: (Bool, Parameters) => Tile, _clock: Clock, _reset: Bool)
```

Comment:
I haven't fully wrapped my head around this Wrapper pattern. Could this be done more cleanly in a  different subclass of Coreplex? Why is this in DefaultCoreplex?